packages/SwingSet/test/test-kernel.js

```diff
@@ -303,13 +303,17 @@ test('outbound call', async t => {
     },
   ]);
 
+  // Move the send to the run-queue
+  await kernel.step();
+  // Deliver the send
```

Comment:
We should test the new functionality here: after the `step()` that moves the send from the acceptance queue to the run-queue, perform a `kernel.dump()` and check that the message now exists in the runQueue (i.e. make a copy of the original code, using `.runQueue`, here). We only need to test this once.